src/com/lftechnology/java/training/niraj/ems/services/implementations/UserServiceImpl.java

```diff
@@ -0,0 +1,507 @@
+        List<Employee> searchResults = employeeDao.findAll(searchCondition, Operators.OR);
+        if (searchResults.size() == 0) {
+            return Constants.USER_NOT_FOUND;
+        }
+        return getFormattedSearchResults(searchResults);
+    }
+
+    /**
+     * Edits user profile
+     * 
+     * @author Niraj Rajbhandari<nirajrajbhandari@lftechnology.com>
+     * @param employee
+     * @param scanner
+     * @param console
+     * @throws SQLException
+     */
+    public void editUserProfile(Employee employee, Scanner scanner, Console console) throws SQLException {
+        List<String> choices = userInput.getMultipleChoices(scanner, console, Constants.EDIT_OPTION_LABEL, 1, 4);
+        Map<String, String> editInfo = getEditInfo(scanner, console, choices);
+        Employee editedEmployee = setAttributes(employee, editInfo);
+        save(editedEmployee);
+    }
+
+    /**
+     * Gets users list
+     * 
+     * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+     * @return
+     * @throws SQLException
+     */
+    public String getUsersList() throws SQLException {
+        Map<String, String> searchCondition = new LinkedHashMap<String, String>();
+        searchCondition.put("status", Status.ACTIVE.getStatus());
+        List<Employee> searchResults = employeeDao.findAll(searchCondition);
+        if (searchResults.size() == 0) {
+            return Constants.USER_NOT_FOUND;
+        }
+        return getFormattedSearchResults(searchResults);
+    }
+
+    /**
+     * Get total users in the db
+     * 
+     * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+     * @return
+     * @throws SQLException
+     */
+    public int count() throws SQLException {
+        return employeeDao.getCount();
+    }
+
+    /**
+     * Terminates or deletes user
+     * 
+     * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+     * @param scanner
+     * @param console
+     * @throws SQLException
+     */
+    public void TerminateDeleteUser(Scanner scanner, Console console, Employee loggedInUser) throws SQLException, CustomException {
+        Map<String, String> condition = new LinkedHashMap<String, String>();
+        condition.put("username", userInput.getInput(scanner, console, Constants.SELECT_USER_EDIT));
+        Employee employee = employeeDao.findByAttributes(condition);
+        if (employee.getId() == null) {
+            LOGGER.warning(Constants.USER_NOT_FOUND);
+        } else {
+            editSelectedUser(scanner, console, employee, loggedInUser);
+        }
+
+    }
+
+    /**
+     * Gets the users choice to terminate or delete a user
+     * 
+     * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+     * @param scanner
+     * @param console
+     * @param employee
+     * @throws SQLException
+     * @throws CustomException
+     */
+    private void editSelectedUser(Scanner scanner, Console console, Employee employee, Employee loggedInUser) throws SQLException,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "console". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1172)